src/tools/pub/cmd/pub-global.md

```diff
@@ -91,8 +88,10 @@ package again.
 ## Running a script
 
 You can run a script from an activated package explicitly using
-`pub global run`, or you can add it to your PATH so that you can run it
-directly at the command line.
+`pub global run`. If the script has been added to the
+`executables` tag in its pubspec file, and the bin directory in
+the cache has been added to your path, you can run the script
```

Comment:
This is a bit complicated. I'd make it a procedure and then use notes to explain why it might fail and what they can do about it.

In particular, give a code example of adding the cache bin directory to the path, even if it's only a Mac example. Or of checking that the path includes the cache bin directory. That way people who only read the code will notice that they need to do something.